lib/AuroraLip/Archives/Formats/PAKb.cs

```diff
@@ -0,0 +1,78 @@
+﻿using AuroraLib.Common;
+using AuroraLib.Common.Struct;
+
+namespace AuroraLib.Archives.Formats
+{
+    public class PAKb : Archive, IHasIdentifier, IFileAccess
+    {
+        public bool CanRead => true;
+
+        public bool CanWrite => false;
+
+        public virtual IIdentifier Identifier => _identifier;
+
+        private static readonly Identifier32 _identifier = new("PAKb");
+
+        public bool IsMatch(Stream stream, in string extension = "")
+            => stream.Match(_identifier);
+
+        protected override void Read(Stream stream)
+        {
+            stream.MatchThrow(_identifier);
+
+            uint file_count = stream.ReadUInt32(Endian.Big);
+
+            FileData[] file_data = new FileData[file_count];
```

Comment:
you can use the struct reader for this in the future.
it's faster and uses less memory if that's your concern ;)

``` c#
FileData[] file_data = stream.Read<FileData>(file_count,Endian.Big);
```

``` c#
 private struct FileData
        {
            public uint crc;
            public uint offset;
            public uint size;
        }
```